packages/waku/src/lib/middleware/dev-server.ts

```diff
@@ -64,6 +59,8 @@ const createStreamPair = (): [Writable, Promise<ReadableStream | null>] => {
   return [writable, promise];
 };
 
+const externalizedWakuClientModules = ['waku/client', 'waku/router/client'];
```

Comment:
I wonder if this limits third-party libraries.
What would happen without `waku/router/client` here?

Reply:
This is really bad, I guess. We shouldn't depend on `waku/router` like this. Otherwise, other router integration like tanstack/router wouldn't be possible.
I guess the solution is to find a dependency from `main.tsx`.

Reply:
Even for `waku/client`, we know the dependency from `main.tsx`, no?